EnglishFilter/sections/general_elemhide.txt

```diff
@@ -97,6 +97,7 @@
 ##.div-gpt-ad:not([style^="width: 1px; height: 1px;"])
 ##[class^="div-gpt-ad"]:not([style^="width: 1px; height: 1px;"])
 !
+##a[href*="//click.soonwe.com/"]
```

Comment:
How many sites require this rule?

Reply:
https://github.com/AdguardTeam/AdguardFilters/pull/88214/commits/9e25e8e21f4f92f824c26c7ea36565775b8b5c6f